scipy/signal/_spectral_py.py

```diff
@@ -57,7 +75,18 @@ def lombscargle(x,
     Raises
     ------
     ValueError
-        If the input arrays `x` and `y` do not have the same shape.
+        If any of the input arrays x, y, freqs, or weights are not 1D. Or, if the input 
+        arrays x, y, and weights do not have the same shape.
+    ZeroDivisionError
+        If the freqs array contains the value 0.
+    ZeroDivisionError
+        If the the sum of the weights array is 0.
+    ValueError
+        If any weight is < 0.
+    TypeError
+        If the normalize parameter is not a bool or str.
+    ValueError
+        If the normalize parameter is not one of the allowed options.
```

Comment:
The `See Also` section is not up to date anymore. I think it would be sufficient to only include ``periodogram``, ``welch`` and ``csd`` there.

Reply:
Will fix.